documentation/release-notes/13.x.x/13.8.0/README.md

```diff
@@ -18,6 +18,10 @@ Property** `targetKey`. The supported `targetKey` values:
 Only the keys listed above are supported. Setting `targetKey` to any other `case:` property is **not supported** and
 will still result in an error.
 
+* **Custom component widgets support for key/value inputs**
+
+Custom Key/Value pairs can be configured in the custom component widgets. These properties are then made accessible inside of the component.
```

Comment:
Can this be explained a bit further? I see that each property key is passed as an input, and if that's true, it should be explained here each key can be acces inside the custom component with an @Input decorator